validphys2/examples/pdf_flavour_plot.yaml

```diff
@@ -1,7 +1,7 @@
 meta:
   title: PDF flavours plot example
-  author: Rosalyn Pearson
-  keywords: [example]
+  author: Rosalyn L. Pearson
```

Comment:
I would honestly change the authors in the example from <any> to `Lazy Person` or `The Creator` or some other funny thing like that. Mainly so that if people uploads a report without changing these fields we know that we don't know who uploaded it instead of thinking it was Rosalyn :__